### PR TITLE
Navy TMD Changes

### PR DESCRIPTION
**Pull request by AugustZilakovs**
completes #4984

### CODE REVIEW

File: units/UES0302/UES0302_unit.bp [hunk 1]
```diff
@@ -856,7 +856,7 @@ UnitBlueprint {
             HeadingArcCenter = 90,
             HeadingArcRange = 100,
             Label = 'RightPhalanxGun01',
-            MaxRadius = 50,
+            MaxRadius = 60,
```
Comment (MrRowey) [inline units/UES0302/UES0302_unit.bp:859]: Wrong Value Should Be 31

File: units/UES0302/UES0302_unit.bp [hunk 2]
```diff
@@ -918,7 +918,7 @@ UnitBlueprint {
             HeadingArcCenter = -90,
             HeadingArcRange = 100,
             Label = 'LeftPhalanxGun01',
-            MaxRadius = 50,
+            MaxRadius = 60,
```
Comment (MrRowey) [inline units/UES0302/UES0302_unit.bp:921]: Same here

File: units/URS0302/URS0302_unit.bp [hunk 3]
```diff
@@ -640,7 +640,7 @@ UnitBlueprint {
             HeadingArcCenter = -90,
             HeadingArcRange = 100,
             Label = 'LeftZapper',
-            MaxRadius = 50,
+            MaxRadius = 60,
```
Comment (MrRowey) [inline units/URS0302/URS0302_unit.bp:643]: Wrong Value

File: units/URS0302/URS0302_unit.bp [hunk 4]
```diff
@@ -700,7 +700,7 @@ UnitBlueprint {
             HeadingArcCenter = 90,
             HeadingArcRange = 100,
             Label = 'RightZapper',
-            MaxRadius = 50,
+            MaxRadius = 60,
```
Comment (MrRowey) [inline units/URS0302/URS0302_unit.bp:703]: Same Here

File: units/XSS0302/XSS0302_unit.bp [hunk 5]
```diff
@@ -512,7 +512,7 @@ UnitBlueprint {
             HeadingArcCenter = 90,
             HeadingArcRange = 100,
             Label = 'AntiMissileLeft',
-            MaxRadius = 50,
+            MaxRadius = 60,
```
Comment (MrRowey) [inline units/XSS0302/XSS0302_unit.bp:515]: Wrong Value

**Review by Tagada14 — APPROVED** [review]

**Pull request merged**